### ISSUE TITLE
Different daily and monthly tariffs for apartments (SRP) rental

### ISSUE THREAD

**Issue by ebayzoev** [WoollyinWalesIT/jomres#138]
It would be very useful to have a feature with an option to select either daily or monthly tariff type for an SRP, like apartment. Currently the workaround is to select different min/max days with different prices, however in the All Listings page the price is shown incorrectly, as for monthly tariff it calculates on per day basis (the lowest start price would be monthly discounted price), while on per day basis it should select the daily tariff. Thus, while monthly tariff selected, the Departure date should be N months ahead (1, 2, 3, ... 12 months), not M days more than 30.

**Comment by ehcanada** [comment]
I agree this would be HUGE! But also for MRPs as well, most properties have weekly and/or monthly discounts.
